en/user_guide/global_page_blocks.rst

```diff
@@ -0,0 +1,23 @@
+Global Pages and Blocks
+=======================
+
+In Open Orchestra, one of the RAD aspects is to simplify the creation of pages by using prefunctionnal blocks.
+These blocks must be configured according to their functionnality (see `Block creation`_ and `Block list`_).
+
+Some blocks could have the same configuration parameters in many pages like header or footer block.
+Global pages are containers allowing to store these globally configured blocks.
+Such blocks are reusable in all pages.
+
+They will appear with blue background in the backoffice right panel whereas the other blocks appear in orange color.
+
+.. image:: ../../images/panel_global_page_blocks.png
+
+The global pages could be viewed through the left menu of the backoffice.
```

Comment:
"viewed through" would mean the menu is transparent. "accessed through" would be more appropriate